lib/spreadsheet_architect/class_methods/xlsx.rb

```diff
@@ -104,7 +104,7 @@ def to_axlsx_package(opts={}, package=nil)
             end
           end
 
-          sheet.add_row row_data, style: styles, types: types
+          sheet.add_row row_data, style: styles, types: types, escape_formulas: options[:escape_formulas]
```

Comment:
Can we make the headers rows use `escape_formulas: options[:escape_formulas]` just the same as the data rows